AirCasting/SessionViews/ShareSession/ShareLocationlessSessionView.swift

```diff
@@ -24,7 +24,7 @@ struct ShareLocationlessSessionView: View {
             .padding()
             .alert(item: $viewModel.alert, content: { $0.makeAlert() })
             .sheet(isPresented: $viewModel.showShareSheet, content: {
-                ActivityViewController(itemsToShare: [viewModel.file as Any]) { activityType, completed, returnedItems, error in
+                ActivityViewController(itemToShare: viewModel.file ?? URL(fileURLWithPath: "")) { activityType, completed, returnedItems, error in
```

Comment:
is silently handling an empty `viewModel.file` good here? Maybe we need in that case?

Reply:
We have a warning when it is empty I think, so will it break anything in this way?

Reply:
Won't break, but won't it be surprising to users? I mean - in this scenario they will be presented with a share sheet for empty URL. Don't we want to show an alert saying "oops, we couldn't fetch share URL for this session"? Or is it 100% guaranteed at this point that viewModel.file will be present?

Reply:
I made a few changes now and as I tested I can say that whenever there is no link there will be an alert presented to the user